samples/Mathematica/TestString.mt

```diff
@@ -0,0 +1 @@
+Test[1"a" <> "b", "ab", TestID -> "Concat \"a\" and \"b\""]
```

Comment:
Dang - I just noticed a copypasta artifact: `Test[1"a" ` should just be `Test["a" `